DEHPMatlab/ViewModel/Row/RowViewModel.cs

```diff
@@ -0,0 +1,73 @@
+﻿// --------------------------------------------------------------------------------------------------------------------
+// <copyright file="RowViewModel.cs" company="RHEA System S.A.">
+// Copyright (c) 2020-2022 RHEA System S.A.
+// 
+// Author: Sam Gerené, Alex Vorobiev, Alexander van Delft, Nathanael Smiechowski, Antoine Théate.
+// 
+// This file is part of DEHPMatlab
+// 
+// The DEHPMatlab is free software; you can redistribute it and/or
+// modify it under the terms of the GNU Lesser General Public
+// License as published by the Free Software Foundation; either
+// version 3 of the License, or (at your option) any later version.
+// 
+// The DEHPMatlab is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+// Lesser General Public License for more details.
+// 
+// You should have received a copy of the GNU Lesser General Public License
+// along with this program; if not, write to the Free Software Foundation,
+// Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
+// </copyright>
+// --------------------------------------------------------------------------------------------------------------------
+
+namespace DEHPMatlab.ViewModel.Row
+{
+    using ReactiveUI;
+
+    /// <summary>
+    /// The <see cref="RowViewModel"/> stores the value of variable from the Matlab Workspace
+    /// </summary>
+    public class RowViewModel: ReactiveObject
+    {
+        /// <summary>
+        /// Backing field for <see cref="Name"/>
+        /// </summary>
+        private string name;
+
+        /// <summary>
+        /// The name of the variable
+        /// </summary>
+        public string Name
+        {
+            get => this.name;
+            set => this.RaiseAndSetIfChanged(ref this.name, value);
+        }
+
+        /// <summary>
+        /// Backing field for <see cref="Value"/>
+        /// </summary>
+        private object value;
```

Comment:
We typically group all privates on top of the file, then constructors then the public props.